docs/guides/v8-upgrade-guide.md

```diff
@@ -80,10 +108,6 @@ InstUI now needs `node.js` version >12.20 to build.
 
 Updated React legacy context to the new context API in the components that use it (e.g. `DrawerLayout`).
 
-##### TreeBrowser
-
-New `TreeBrowser.Node` component for rendering the `renderBeforeItems` and `renderAfterItems` in the `TreeBrowser` (Also available in 7.5.0).
-
```

Comment:
I've removed this because we are not listing here every new feature addition in v8